lms/djangoapps/instructor/views/api.py

```diff
@@ -3275,8 +3269,8 @@ def re_validate_certificate(request, course_key, generated_certificate):
         certificate_invalidation.deactivate()
 
     # We need to generate certificate only for a single student here
-    students = [certificate_invalidation.generated_certificate.user]
-    instructor_task.api.generate_certificates_for_students(request, course_key, students=students)
+    student = [certificate_invalidation.generated_certificate.user]
+    instructor_task.api.generate_certificates_for_students(request, course_key, specific_student=student)
```

Comment:
see my comments below in tasks_helper.py. It doesn't make sense to do it this way because it introduces a lot of confusion